src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/OpprettBehandling/useOpprettBehandling.ts

```diff
@@ -145,11 +168,16 @@ const useOpprettBehandling = (
                             behandlingÅrsak: behandlingsårsak.verdi as BehandlingÅrsak,
                             navident: innloggetSaksbehandler?.navIdent,
                             nyMigreringsdato:
+                                skjema.felter.behandlingstype.verdi ===
+                                Behandlingstype.MIGRERING_FRA_INFOTRYGD
+                                    ? skjema.felter.migreringsdato.verdi
+                                    : undefined,
+                            barnasIdenter:
                                 skjema.felter.behandlingstype.verdi ===
                                     Behandlingstype.MIGRERING_FRA_INFOTRYGD &&
                                 skjema.felter.behandlingsårsak.verdi ===
-                                    BehandlingÅrsak.ENDRE_MIGRERINGSDATO
-                                    ? skjema.felter.migreringsdato.verdi
+                                    BehandlingÅrsak.HELMANUELL_MIGRERING
```

Comment:
Kan man opprette en metode erHelmanuellMigrering for å bedre lesbarehet og for at den brukes på flere plasser (OpprettBehandlingValg)

Reply:
Ein metode i useOpprettBehandling kan ikkje gjenbrukes i OpprettBehandlingValg. OpprettBehandlingValg brukes av både OpprettBehandling og KnyttTilBehandling (journalføring).

Reply:
ok